EasyPost/Properties/AssemblyInfo.cs

```diff
@@ -15,7 +15,12 @@
 [assembly: AssemblyCulture("")]
 
 // Make "private" methods testable.
-[assembly: InternalsVisibleTo("EasyPostTest, PublicKey=0024000004800000940000000602000000240000525341310004000001000100b9d909875738653bb31cbe8e83089f3b748eaaed7fbf54fd6601f73803a24005cac0cea99143a6ef6183a98cdeda360c198fb1d72c130c1de6dd7475c9cc7c01cb9fe2dae674b9531e6d431f8343496f023f356919c007f155254d1862f1734c2ba73441e252dc1dbd7a9b6e4fe048086e12d8742db955894d89685bb1bfb5c1")]
+//[assembly: InternalsVisibleTo("EasyPostTest, PublicKey=0024000004800000940000000602000000240000525341310004000001000100b9d909875738653bb31cbe8e83089f3b748eaaed7fbf54fd6601f73803a24005cac0cea99143a6ef6183a98cdeda360c198fb1d72c130c1de6dd7475c9cc7c01cb9fe2dae674b9531e6d431f8343496f023f356919c007f155254d1862f1734c2ba73441e252dc1dbd7a9b6e4fe048086e12d8742db955894d89685bb1bfb5c1")]
```

Comment:
Why is this being commented out? Could we leave a small comment in code to describe why?